webui/tests/cypress/support/commands.js

```diff
@@ -14,6 +14,13 @@ const saltAndHashPassword = password => {
   return sha512(passwordSalt + password);
 };
 
+Cypress.on('uncaught:exception', (err, runnable) => {
```

Comment:
the warnings I think might be retrievable, but they are not helpful in any way. They just say `uncaught exception` of `undefined` and there's nothing else that provides any clues as to what. It just blames React for generating it. I decided to even remove the warnings because they did not give any hints as to what causes them and how to "fix" it.